roles/mariadb/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
-mariadb_keyserver: keyserver.ubuntu.com
+mariadb_keyserver: "hkp://keyserver.ubuntu.com:80"
```

Comment:
I switched this back to non-hkps because 16.04 doesn't support it. I'm fine with keeping this "non-secure" version since the official MariaDB instructions use this as well. It makes it way easier for people to stay on 16.04 for now.